Extension/package.json

```diff
@@ -2361,6 +2361,37 @@
           "description": "%c_cpp.configuration.doxygen.generatedStyle.description%",
           "scope": "resource"
         },
+        "C_Cpp.doxygen.selectDoxygenTags": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "attention",
+              "author",
+              "authors",
+              "bug",
+              "copyright",
+              "date",
+              "details",
+              "exception",
+              "invariant",
+              "note",
+              "pre",
+              "post",
+              "remark",
+              "remarks",
+              "result",
+              "return",
+              "since",
+              "test",
+              "todo",
+              "version",
+              "warning"
+              ],
+          "description": "%c_cpp.configuration.doxygen.selectDoxygenTags.description%",
+          "scope": "resource"
```

Comment:
This should have a default value that matches the current set of hard-coded tags in the extension today.

Something like:
```
  "default": [
    "brief",
    "tparam",
    "param",
    "return",
    "retval",
    "deprecated",
    "exception"
  ]
```